### ISSUE TITLE
Narrator doesn't read contents of accessible view for terminal

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#196422]
Testing #196124

1. Turn on Narrator
2. Enable Screen Reader Optimized mode by running the Toggle Screen Reader Accessibility Mode command
3. Open a terminal 
4. Run some commands
5. Open the accessible view
6. :bug: I hear "the editor is not accessible at this time" and no content from the view is read

This repros regardless of whether I have the accessible view hidden

**Comment by joyceerhl** [comment]
I think this is because I had `editor.accessibilitySupport` set to `off`, but I would have expected that running the toggle command updated the setting's value, especially because the status bar item saying Screen Reader Optimized appeared

**Comment by joyceerhl** [comment]
Wait, I removed that setting and I still hear "the editor is not accessible at this time" when I open the accessible view

**Comment by meganrogge** [comment]
Please try with NVDA/ Jaws. There are sometimes issues with Narrator

**Comment by meganrogge** [comment]
@joyceerhl , I reverted a change I had made to the screen reader optimized mode handling. Could you pls see if you can reproduce?

**Comment by joyceerhl** [comment]
Cannot repro anymore, actioning that notification now appropriately updates the value of `"editor.accessibilitySupport"`
